app/models/account.rb

```diff
@@ -56,13 +55,7 @@ def self.cache_or_create_by!(user)
   def self.create_if_needed!(user)
     attrs = account_attrs_from_user(user)
 
-    accts = where(idme_uuid: attrs[:idme_uuid])
-            .where.not(idme_uuid: nil)
-            .or(
-              where(sec_id: attrs[:sec_id])
-              .where.not(sec_id: nil)
-            )
-
+    accts = where(uuid: user.uuid)
```

Comment:
Wait I don't think this is right - you're trying to match up the (idme_uuid or secid_uuid) with the (VA.gov-generated) account_id

Reply:
And also since we've been gradually adding secids to the accounts table for the last  few weeks, need to make sure we're querying by the raw secid, not the secid_uuid.